modules/test-runner/src/store/store.test.ts

```diff
@@ -12,11 +12,15 @@ import {
   TEST_STORE_CONDITIONAL_COMMITMENT,
 } from "../util";
 
+export const storeTypesNoPostgres = Object.keys(StoreTypes).filter(
+  store => store !== StoreTypes.Postgres,
+);
+
```

Comment:
is there a way to test this with a postgres store?

Reply:
Yes, I figured it out, will add this to the suite.